src/components/Toggle/Toggle.jsx

```diff
@@ -33,7 +33,9 @@ class Toggle extends React.Component<Props> {
     onChange: () => {},
   };
   onClick(toggle: Object, index: number) {
-    this.props.onChange(toggle.label, index);
+    if (toggle.disabled !== true) {
```

Comment:
I think you can write just `!toggle.disabled`.

What's `toggle`, by the way? Could you maybe change the annotation while you're here?